Find a functionally equivalent code:android.os.Bundle.getStringArrayList:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the value associated with the given key, or null if
 * no mapping of the desired type exists for the given key or a null
 * value is explicitly associated with the key.
 *
 * @param key a String, or null
 * @return an ArrayList<String> value, or null
 */

@@ -1,13 +1,3 @@
 {
-    unparcel();
-    Object o = mMap.get(key);
-    if (o == null) {
-        return null;
-    }
-    try {
-        return (ArrayList<String>) o;
-    } catch (ClassCastException e) {
-        typeWarning(key, o, "ArrayList<String>", e);
-        return null;
-    }
+    return super.getStringArrayList(key);
 }

